python/lsst/meas/base/forcedPhotImage.py

```diff
@@ -25,10 +25,10 @@
 `ForcedPhotCcdTask`, `ForcedPhotCoaddTask`).
 """
 
-import lsst.afw.table
+import lsst.afw.table as afwTable
```

Comment:
Ditto on https://developer.lsst.io/python/style.html#import-statements-should-follow-a-standard-pattern